contentcuration/contentcuration/models.py

```diff
@@ -564,9 +564,12 @@ def save(self, *args, **kwargs):  # noqa: C901
 
     def get_thumbnail(self):
         if self.thumbnail_encoding:
-            thumbnail_data = ast.literal_eval(self.thumbnail_encoding)
-            if thumbnail_data.get("base64"):
-                return thumbnail_data["base64"]
+            thumbnail_data = load_json_string(self.thumbnail_encoding)
```

Comment:
Got invalid json errors on this, so use custom json loader instead of ast.literal_eval for security reasons